public/css/app.css

```diff
@@ -1,85 +1,73 @@
-html,
-body {
+
+
+html, body {
```

Comment:
We merged a Stylelint PR with some refactored CSS earlier today. We may need to rebase this against **master**.